iac/cal-itp-data-infra/logging/us/gtfs_rt_alert.tf

```diff
@@ -0,0 +1,37 @@
+resource "google_monitoring_notification_channel" "email" {
+  type         = "email"
+  display_name = "GTFS-RT Archivers Email Alert Channel"
+  description  = "Send gtfs-rt archivers alerts to email."
+  labels = {
+    email_address = "dds.app.notify@dot.ca.gov"
+  }
+}
+
+resource "google_monitoring_alert_policy" "gtfs_rt_bucket_writes_alert" {
+  display_name          = "GTFS-RT Bucket Writes Alert"
+  combiner              = "OR"
+  notification_channels = [google_monitoring_notification_channel.email.name]
+  conditions {
+    display_name = "Less than 1500 writes to specific bucket in 15 minutes"
+    condition_sql {
+      query = <<-QUERY
+      SELECT count(*) AS row_count FROM `projects/your-project-id/logs/cloudaudit.googleapis.com%2Fdata_access`
```

Comment:
does this need to be the actual project id? We have two: `cal-itp-data-infra` and `1005246706141`. If you try to `terraform apply` locally, which one does it accept?